### DISCUSSION TITLE
[Bug]: No readme description

### DISCUSSION THREAD

**Discussion by coolassassin** [npm/feedback#787] [General]
Last several months i catch issue with readme package render on package page.
This is my package:
https://www.npmjs.com/package/reactcci
This package has readme. And you probably see it by the link or maybe not.
It could show or not in random time.
Same version of package bellow in different time:
![telegram-cloud-photo-size-2-5348442071729553830-y](https://user-images.githubusercontent.com/17389477/195415835-79b1c3d2-3e5a-41fd-88f0-da49c8475c07.jpg)
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/17389477/195415889-c970fa3f-4dd4-411e-a67a-91ab17a8a159.png">
It can be like that, on any device.

Should i change something in package.json to make it stable?
I probably lose some users because of it.

**Comment by yilmazdurmaz** [comment]
I stumbled upon this today. Some package I visited yesterday with full descriptive README was showing empty today.

caching issue? I am not sure. if it was cached, I would have seen the cached README from my yesterday's visit.

To get past the issue, I opened developer tools so that I could use the "Hard Reset" refresh on the page. (right click on the refresh button while dev-tools is open).

But this does not solve the main issue: Caching or not, npm package pages fail to load README files.

**Reply by coolassassin** [reply]
Yeah. More often, i see this issue on phone and there i have no tools to reset cache, same as on computer
